sport/app/rugby/feed/OptaFeed.scala

```diff
@@ -18,9 +18,16 @@ case object WorldCup2015 extends OptaEvent {
   override def hasGroupTable(stage: Stage.Value): Boolean = stage == Stage.Group
 }
 
+case object WorldCup2019 extends OptaEvent {
```

Comment:
can this entire file be deleted?

Reply:
Yes! I need to do more testing but yeah I'm not even sure we have an API key that's valid for Opta rugby anymore.

Reply:
👍 We simply throw an error in the console on old rugby matches, currently.

Reply:
cool thanks @rcrphillips